### ISSUE TITLE
A11y_SQL Azure Data Studio_New Terminal_Keyboard:Tab key or shift+tab does not move to link "https://aka.ms/pscore6" is not working. 

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/azuredatastudio#22333]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev1;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#Benchmark;#Keyboard;#AINotInScope;#AILimited;#LinkedBug;#Linked:182849;#External;#External:VSCode;

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.45.0-insider (user setup)
Windows Version: Windows11

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to view menu item in the menu bar and use down key arrow to select item "terminal"
4. Use shift+tab to navigate to https://aka.ms/pscore6

### Actual:
On navigating using shift+tab or tab to link https://aka.ms/pscore6 keyboard key tab cannot access it.

### Expected:
Tab and shift+tab from the first line of command should move to link https://aka.ms/pscore6

### User Impact:
Keyboard users will not be able to access the link.

### Attachment:
[22333_A11y_SQL.Azure.Data.Studio_New.Terminal_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/10979474/22333_A11y_SQL.Azure.Data.Studio_New.Terminal_Keyboard.zip)

**Comment by Roopesh-T** [comment]
Opening this bug as per @alanrenmsft comment in https://github.com/microsoft/azuredatastudio/issues/12984

**Comment by alanrenmsft** [comment]
@Roopesh-T Could you please open an issue in [VSCode repo](https://github.com/microsoft/vscode), this is an issue we inherited from VSCode.
![terminal](https://github.com/microsoft/azuredatastudio/assets/13777222/a6f09e37-5c67-4d98-a732-e62755df237f)

**Comment by Roopesh-T** [comment]
Logged External bug:https://github.com/microsoft/vscode/issues/182849

**Comment by alanrenmsft** [comment]
@Roopesh-T VSCode team has closed the external bug and provided a way to access the links, please refer to https://github.com/microsoft/vscode/issues/182849#issuecomment-1553560118.

Please test it in ADS and close the issue.
